src/components/o-card/_template.js

```diff
@@ -0,0 +1,14 @@
+import html from 'nanohtml';
+
+export default (props, childrenFragment) => html`
+  <article class="o-card__wrap">
+    
+    <div class="o-card__side o-card__side--1" name="side-1">
```

Comment:
name is not valid here: https://www.w3schools.com/tags/att_name.asp please use `data-name`